UIKitDz/UIKitDz/ViewController.swift

```diff
@@ -7,8 +7,127 @@
+        label.text = "Результат: "
+        return label
+    }()
+    private lazy var findNumberButton: UIButton = {
+        let button = UIButton(type: .custom)
+        button.backgroundColor = .gray
+        button.layer.cornerRadius = 15
+        button.setTitle("Игра \"Угадай число\"", for: .normal)
+        button.titleLabel?.textAlignment = .center
+        button.frame = CGRect(x: 20, y: 240, width: 140, height: 70)
+        button.addTarget(self, action: #selector(findNumber(paramTarget:)), for: .touchUpInside)
+        return button
+    }()
+    private lazy var findNumberTextField: UITextField = {
+        let textField = UITextField()
+        textField.placeholder = "Введите число от 0 до 100"
+        textField.frame = CGRect(x: 20, y: 320, width: 250, height: 80)
+        textField.isHidden = true
+        return textField
+    }()
+    private lazy var tryButton: UIButton = {
+        let button =  UIButton(configuration: .filled())
+        button.setTitle("Применить", for: .normal)
+        button.frame = CGRect(x: 20, y: 400, width: 140, height: 40)
+        button.isHidden = true
+        button.addTarget(self, action: #selector(tryToFindNumber(paramTarget:)), for: .touchUpInside)
+        return button
+    }()
     override func viewDidLoad() {
         super.viewDidLoad()
+        setupUI()
+    }
+    override func viewDidAppear(_ animated: Bool) {
+        alertGreetings()
+    }
+    func setupUI() {
+        view.addSubview(notification)
+        view.addSubview(nameLabel)
+        view.addSubview(additionButton)
+        view.addSubview(resultLabel)
+        view.addSubview(findNumberButton)
+        view.addSubview(findNumberTextField)
+        view.addSubview(tryButton)
+    }
+    func alertGreetings() {
+        let greetingAlertController = UIAlertController(title: "Привет",
+                                                        message: "Введи своё имя",
+                                                        preferredStyle: .alert)
+        let okAlertAction = UIAlertAction(title: "ok", style: .default) { _ in
+            self.nameLabel.text? += greetingAlertController.textFields?.first?.text ?? ""
+        }
+        greetingAlertController.addTextField()
+        greetingAlertController.addAction(okAlertAction)
+        self.present(greetingAlertController, animated: true)
+    }
+    @objc func addition(paramTarget: UIButton) {
+        let additionAlertController = UIAlertController(title: "Сложение",
+                                                        message: "Введи 2 значения",
+                                                        preferredStyle: .alert)
+        additionAlertController.addTextField()
+        additionAlertController.addTextField()
+        let resultAlertAction = UIAlertAction(title: "Результат", style: .default) { _ in
+            guard let firstTerm = Int(additionAlertController.textFields?[0].text ?? "") else { return }
```

Comment:
```suggestion
            guard 
               let firstTerm = Int(additionAlertController.textFields?[0].text ?? "") 
               let secondTerm = Int(additionAlertController.textFields?[1].text ?? "") 
            else {
               return 
             }
``` 
и additionAlertController.textFields?[0]  безопасно?

Reply:
Да? там же вопросики